### PR TITLE
normalize uri, src_path and dst_path in reader-common

### PR DESCRIPTION
**Pull request by mchataigner**
This way, all ready will have same normalized URI.

### CODE REVIEW

File: readers/elasticsearch/src/test/java/com/criteo/hadoop/garmadon/elasticsearch/UriHelperTest.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.criteo.hadoop.garmadon.elasticsearch;
```
Comment (ashangit) [inline readers/elasticsearch/src/test/java/com/criteo/hadoop/garmadon/elasticsearch/UriHelperTest.java]: This file should be moved to readers common tests

**Comment by ashangit** [comment]
Could you also update Readme section "HDFS URI standardization" to specifiy that the hdfs-mapping file should be added to conf folder of any garmadon readers

File: readers/elasticsearch/src/test/java/com/criteo/hadoop/garmadon/elasticsearch/UriHelperTest.java (hunk 1 above)
Comment (mchataigner) [reply readers/elasticsearch/src/test/java/com/criteo/hadoop/garmadon/elasticsearch/UriHelperTest.java]: done

File: Readme.md [hunk 2]
```diff
@@ -128,6 +128,14 @@ Here is the event repartition on ElasticSearch cluster over a 4h period
 
 ![](doc/images/es/event_type_repartition.png)
 
+### Common reader configuration
```
Comment (ashangit) [inline Readme.md]: Please just update the section HDFS URI standardization (not create a new one)
To clarify on which component it should be added

Comment (mchataigner) [reply Readme.md]: done

**Pull request merged**